.travis.yml

```diff
@@ -5,7 +5,7 @@ language: scala
 services:
   - docker
 
-# This version will be also tagged as 'latest'
+# This release is experimental only, previous version is remains 'latest'
```

Comment:
According to https://kafka.apache.org/downloads: `2.8.0 is the latest release. The current stable version is 2.8.0.`  so this comment can be reverted

Reply:
Fixed !
@wurstmeister kindly review this PR. Support for KRaft Mode without zookeeper should/will be a separate PR.